charts/victoria-metrics-alert/CHANGELOG.md

```diff
@@ -2,7 +2,7 @@
 
 ## Next release
 
-- TODO
+- Fix vmalert notifier address if builtin alertmanager is enabled and using baseURLPrefix.
```

Comment:
Could you please provide the link to this PR

Reply:
Ops, missed this, will add in next one.